<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TourWidget</class>
 <widget class="QWidget" name="TourWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tour</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBar" name="m_toolBarControl">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <addaction name="m_actionNewTour"/>
     <addaction name="separator"/>
     <addaction name="m_actionOpenTour"/>
     <addaction name="separator"/>
     <addaction name="m_actionSaveTour"/>
     <addaction name="m_actionSaveTourAs"/>
     <addaction name="separator"/>
     <addaction name="m_actionAddFlyTo"/>
     <addaction name="separator"/>
     <addaction name="m_actionMoveUp"/>
     <addaction name="m_actionMoveDown"/>
     <addaction name="separator"/>
     <addaction name="m_actionDelete"/>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="m_listView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="m_toolBarPlayback">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <addaction name="actionPlay"/>
     <addaction name="separator"/>
     <addaction name="actionPause"/>
     <addaction name="separator"/>
     <addaction name="actionStop"/>
    </widget>
   </item>
  </layout>
  <action name="m_actionAddFlyTo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/list-add.png</normaloff>:/marble/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add FlyTo</string>
   </property>
  </action>
  <action name="m_actionMoveUp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/go-up.png</normaloff>:/marble/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Move up</string>
   </property>
   <property name="toolTip">
    <string>Move up</string>
   </property>
  </action>
  <action name="m_actionMoveDown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/go-down.png</normaloff>:/marble/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Move down</string>
   </property>
  </action>
  <action name="m_actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/list-remove.png</normaloff>:/marble/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="m_actionNewTour">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/document-new.png</normaloff>:/marble/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New Tour</string>
   </property>
  </action>
  <action name="m_actionSaveTour">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/document-save.png</normaloff>:/marble/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Tour</string>
   </property>
  </action>
  <action name="m_actionSaveTourAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/document-save-as.png</normaloff>:/marble/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save Tour As</string>
   </property>
  </action>
  <action name="m_actionOpenTour">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/document-open.png</normaloff>:/marble/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Tour</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/playback-play.png</normaloff>:/marble/playback-play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Begins touring</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/playback-pause.png</normaloff>:/marble/playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pauses touring until you start it again</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/marble/playback-stop.png</normaloff>:/marble/playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stops touring (playing again will start tour from the start)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="libmarble.qrc"/>
 </resources>
 <connections/>
</ui>
